wled00/const.h

```diff
@@ -578,8 +578,14 @@
 #else
   #define DEFAULT_LED_PIN 16   // aligns with GPIO2 (D4) on Wemos D1 mini32 compatible boards (if it is unusable it will be reassigned in WS2812FX::finalizeInit())
 #endif
-#define DEFAULT_LED_TYPE TYPE_WS2812_RGB
-#define DEFAULT_LED_COUNT 30
+
+#ifndef DEFAULT_LED_TYPE
+  #define DEFAULT_LED_TYPE TYPE_WS2812_RGB
+#endif
+
+#ifndef DEFAULT_LED_COUNT
+  #define DEFAULT_LED_COUNT 30
+#endif
 
```

Comment:
aehem .... @ingDIY  you know that this part was just rejected as a PR. 

Please remove unrelated changes to keep the PR scope clean.

Reply:
yes, but I was working on PR on different branches. I did something wrong managing the branches.
should I pull-request from the beginning?

Reply:
> yes, please start fresh. It maybe best to fork WLED again for having a fresh repo, then making a brach for your PR that just contains the code changes.
> 
> See also:
> 
> * [How to properly submit a PR](https://github.com/Aircoookie/WLED/wiki/How-to-properly-submit-a-PR)
> * https://github.com/Aircoookie/WLED/blob/0_15/CONTRIBUTING.md

ok, i strarted from scratch and made a new PR